CMakeLists.txt

```diff
@@ -296,6 +309,13 @@ add_executable(ufs_model NEMS/src/MAIN_NEMS.F90)
 add_dependencies(ufs_model ufs)
 target_link_libraries(ufs_model ufs esmf w3nco::w3nco_d)
 
+if(S2S OR CMEPS OR CDEPS)
```

Comment:
Why `S2S`?   `S2S` is an application.  It enables/disables `CMEPS` and `CDEPS`.  If `S2S` is `ON`, `CMEPS` is `ON`.
If `S2S` is `OFF`, but any of the data components are ON, `CDEPS` is `ON`.

Reply:
Yes, I think the S2S can be removed in here because S2S app was defining CMEPS. Right? In the old version of the build S2S was a shortcut and was not setting CMEPS but this is changes after fixing the build system. @BinLiu-NOAA could you try to test this by removing S2S in your branch. I am not expecting any issue.

Reply:
@aerorahul and @uturuncoglu, sure I will take out S2S here and commit it back to the feature/hafs_sync_202107 branch. And will conduct regression tests after it being synced with ufs-weather-model develop again this afternoon.

Reply:
This has been removed and tests went through.